packages/cms-data-sync/src/utils/rich-text.ts

```diff
@@ -34,7 +35,26 @@ export const convertHtmlToContentfulFormat = (html: string) => {
   // an important part of rich text input anyway, so we
   // can just remove them here
   const htmlWithoutDivTag = html.replace(/<[\\/]{0,1}(div)[^><]*>/g, '');
+  verboseLog(`HTML pre-parsed:\n${html}`);
+  verboseLog(`HTML post-parsed:\n${html}`);
```

Comment:
should the post-parsed be `htmlWithoutDivTag`?

Reply:
Good spot.